.prettierignore

```diff
@@ -0,0 +1,5 @@
+src/images/icons.js
```

Comment:
maybe glob pattern `src/images/**/*.js`?

Reply:
@coderitual 
hmm why not, but I'm thinking of someone manually adding script there for images manipulation will miss prettier formatting? I'm not sure if this can really happen?

Reply:
probably it means that `src/images/` is not the best folder to keep generated js files there